integration-test/test.bats

```diff
@@ -101,22 +130,35 @@
 
 @test "govvv build - reads Version from ./VERSION file" {
     tmp="${BATS_TMPDIR}/a.out"
-    run bash -c "cd ${BATS_TEST_DIRNAME}/app-versioned && govvv build -o ${tmp} ."
+    run bash -c "cd ${BATS_TEST_DIRNAME}/app-versioned && govvv build -o ${tmp}"
     echo "$output"
     [ "$status" -eq 0 ]
 
     run "$tmp"
     echo "$output"
     [ "$status" -eq 0 ]
-    [[ "$output" == "Version=2.0.1" ]]
+    [[ "$output" == "Version=2.0.1-app-versioned" ]]
+}
+
+@test "govvv build - ./VERSION file overridden by -version option" {
+    tmp="${BATS_TMPDIR}/a.out"
+    run bash -c "cd ${BATS_TEST_DIRNAME}/app-versioned && govvv build -o ${tmp} -version 1.2.3-command-line"
+    echo "$output"
+    [ "$status" -eq 0 ]
+
+    run "$tmp"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    [[ "$output" == "Version=1.2.3-command-line" ]]
 }
 
 @test "govvv compiled with govvv" {
-    run govvv install -a
+    run govvv build
```

Comment:
please don't change this. `go install -a` does something differently by recompiling all packages with no cache. this helps govvv compile govvv.

Reply:
It rebuilds the standard runtime etc.  Not a big deal on a travis instance, but won't work on my MBP.  I suspect you have R/W access to your `GOROOT`.

Reply:
@maguro try this:

```
$ git clone https://github.com/ahmetb/govvv ~/workspace/gopath-govvv/src/github.com/ahmetb/govvv
$ export GOPATH=~/workspace/gopath-govvv
$ cd $GOPATH/src/github.com/ahmetb/govvv
$ go install -a
$ which govvv
```

Reply:
I get:

    $ cd $GOPATH/src/github.com/ahmetb/govvv
    $ go install -a
    go install runtime/internal/sys: open /usr/local/go/pkg/darwin_amd64/runtime/internal/sys.a: permission denied
    $ echo $?
    1

Reply:
According to [golang issue 6424](https://github.com/golang/go/issues/6424#issuecomment-228081889):

> Running `go install -a` means to install all the packages, including the standard library. It's usually not what you want to run.

Reply:
@maguro I guess we can do following then:

```
go clean
go install
```

this was govvv still be in $PATH (contains $GOPATH/bin), so `run govvv` will keep working and you won't need `rm ./govvv`. do you mind trying this out?

Reply:
A new `govvv`, built by `govvv`, is not created; which is what I thought the test was attempting to do.

AFAICS, the test is superfluous in the first place.

Reply:
FYI, I tried something different that does not use `govvv build`.  I think that it's more in tune with the test's original intention.

Reply:
I don't mean to be a pest but I'm eager to get my version option patch in.  TIA.  :)